test/unit_tests/aievec_tests/bf16_add_reduce/bf16_add_reduce.mlir

```diff
@@ -1,8 +1,8 @@
 // RUN: aie-opt %s -affine-super-vectorize="virtual-vector-size=16 test-fastest-varying=0 vectorize-reductions=true" --convert-vector-to-aievec="aie-target=aieml" -lower-affine | aie-translate -aieml=true --aievec-to-cpp -o dut.cc
 // RUN: xchesscc_wrapper aie2 -f -g +s +w work +o work -I%S -I. %S/testbench.cc dut.cc
 // RUN: mkdir -p data
-// RUN: xme_ca_udm_dbg -qf -T -P %aietools/data/aie_ml/lib/ -t "%S/../profiling.tcl ./work/a.out" >& xme_ca_udm_dbg.stdout
-// RUN: FileCheck --input-file=./xme_ca_udm_dbg.stdout %s
+// RUN: xca_udm_dbg --aiearch aie-ml -qf -T -P %aietools/data/aie_ml/lib/ -t "%S/../profiling.tcl ./work/a.out" >& xca_udm_dbg --aiearch aie-ml.stdout
+// RUN: FileCheck --input-file=./xca_udm_dbg --aiearch aie-ml.stdout %s
```

Comment:
Oh cool!  I didn't know that worked.  Is this the preferred way of running the aie2 simulator?

Reply:
xme_ca_udm_dbg is obsolete today. xca_udm_dbg --aiearch aie-ml is a preferred way of running the aie2 simulator now.

Reply:
With which version of Vitis does `xca_udm_dbg --aiearch aie-ml` start? I'm seeing errors related to this in my builds.

Reply:
daily_latest

Reply:
for 2022.2 use xme_ca_udm_dbg;
for 2023.1 and later use xca_udb_dbg --aiearch aie-ml